src/libs/ajax.js

```diff
@@ -96,53 +97,25 @@ const Cromwell = signal => ({
   }
 })
 
+// TODO: how to get these values?
+const wdsInstanceId = '15f36863-30a5-4cab-91f7-52be439f1175'
+const wdsApiVersion = 'v0.2'
```

Comment:
we'll need to decide how to get these in here...

Reply:
I'd suggest making a ticket. I think we'll need Leo to provide this and render it into a config file, and also do the same in CBAS